project.md

```diff
@@ -11,6 +11,7 @@ By default, the Telemetry app is designed to work along with [GLPI](http://glpi-
   * [Footer links](#footer-links)
   * [Templates contents](#templates-contents)
   * [Dynamic references](#dynamic-references)
+* [Database adaptation](#database-adaptation)
```

Comment:
missing spaces ?

Reply:
No, this is at the same level as Json and User interface ;)